### PR TITLE
fix: automation should use conventional commits

### PR DESCRIPTION
**Pull request by jeremylong**
While I'm not perfect with this - I've been trying to follow conventional commits: https://www.conventionalcommits.org/en/v1.0.0/

This PR updates the release plugin to use conventional commit prefix when making commits during the release process.

### CODE REVIEW

**Review by aikebah — APPROVED** [review]

**Comment by aikebah** [comment]
Maybe reword the PR to "automation should use conventional commits" as the PR concerns both mvn release and the fp gitops commits

**Review by aikebah — APPROVED** [review]

**Comment by jeremylong** [comment]
Thanks for pointing that out - the gitops was (obviously) added after I created the PR...
